src/sortStudents.ts

```diff
@@ -1,16 +1,60 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+)
+  : Student[] {
+  let result: Student[] = Object.assign([], students);
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  if (sortBy === 'name' || sortBy === 'surname') {
```

Comment:
```suggestion
  if (sortBy === SortType.Name || sortBy === SortType.Surname) {
```
and you can use it like here everywhere